src/Acme/UserBundle/Repository/UserRepository.php

```diff
@@ -0,0 +1,15 @@
+<?php
```

Comment:
True that, Thanks! I think it was created automatically when i used the command line to create the entity and I overlooked it.